frontend/src/pages/Contracts/Contract/ContractDetails/fields.tsx

```diff
@@ -132,6 +133,7 @@ export const editFields: IEditFields[] = [
     fieldLabel: "Subcontractors",
     fieldName: "subcontractor_id",
     fieldType: "multiselect",
+    pickerName: "subcontractor_option",
     tableName: "resource",
```

Comment:
remove tableName as not required.

Reply:
Removed all the tableNames you mentioned.